src/containers/Layout/MainLayout.tsx

```diff
@@ -61,6 +74,15 @@ function MainLayout({ title = '', isHeader, children }: Props) {
 
   return (
     <MainLayoutWrapper>
+      <div className="ust-warning">
+        <p>
+          <span role="img" aria-label="warning">
+            ⚠️
+          </span>{' '}
+          Wormhole UST on Venus is not recognized by CEXs. Please convert it to WrappedUST before
+          using.
```

Comment:
```suggestion
          Wormhole UST on Venus is <strong>not</strong> recognized by CEXs. Please swap it to WrappedUST
          before depositing on a CEX.
```

Reply:
Just pushed an update for this:
<img width="1243" alt="Screenshot 2022-03-25 at 17 52 53" src="https://user-images.githubusercontent.com/44675210/160175410-9f7b119a-988f-44c5-b728-e8bef9f60bbc.png">